acceptance/bundle/run/spark_python_task/databricks.yml.tmpl

```diff
@@ -0,0 +1,24 @@
+bundle:
```

Comment:
Is this exact copy of deploy/cluster/simple/databricks.yml.tmpl?

In that case, can we place these two tests close together, put databricks.yml.tmpl in the common parent and then do something like this to avoid copying?

`envsubst < $TESTDIR/../databricks.yml.tmpl > databricks.yml`

Reply:
We can, but I'd rather optimize for separation of concerns rather than for template re-use in this style of acceptance tests.

These tests are covering two different use cases (deploy a cluster and run a task), even though they were sharing the same template in the test when written in go. Additionally, I'd like to think of these acceptance tests self-contained rather than having dependencies on each other and thus would prefer the tests to have their own templates. Thoughts?

Reply:
I'll move these tests to [acceptance](https://github.com/databricks/cli/tree/main/acceptance)/[bundle](https://github.com/databricks/cli/tree/main/acceptance/bundle) folder